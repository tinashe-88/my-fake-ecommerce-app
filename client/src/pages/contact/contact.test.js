import React from 'react'
import { shallow } from 'enzyme'

import ContactPage from './contact'

it('should render Homepage component', () => {
  expect(shallow(<ContactPage />)).toMatchSnapshot()
})