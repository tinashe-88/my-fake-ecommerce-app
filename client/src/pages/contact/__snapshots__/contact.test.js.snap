// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Homepage component 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <styled.h2>
        Let's connect
      </styled.h2>
      <styled.div>
        <Styled(styled.div)>
          <TwitterIcon />
        </Styled(styled.div)>
        <Styled(styled.div)>
          <InstagramIcon />
        </Styled(styled.div)>
        <Styled(styled.div)>
          <GithubIcon />
        </Styled(styled.div)>
      </styled.div>
    </styled.div>
  </styled.div>
</styled.div>
`;
